helper/schema/schema_test.go

```diff
@@ -3947,6 +3947,33 @@ func TestSchemaMap_Validate(t *testing.T) {
 			Err: false,
 		},
 
+		"Good sub-resource, interpolated value": {
```

Comment:
IsComputed basically checks if the value is known during validation, i.e. it's in the config and doesn't contain any UnknownValues.   If you look at the source to `Get`, it's actually going to call `IsComputed` there too, and otherwise return `GetRaw`. 

I think the PR is still good on its own, so I can merge this and I'll send a follow up PR with the additional changes.